src/Resources/contao/drivers/DC_Folder.php

```diff
@@ -2692,12 +2740,28 @@ protected function generateTree($path, $intMargin, $mount=false, $blnProtected=t
 				{
 					$return .= ' <a href="'.$this->addToUrl('&amp;act=move&amp;mode=2&amp;pid='.$currentEncoded).'" title="'.\StringUtil::specialchars(sprintf($GLOBALS['TL_LANG']['tl_files']['uploadFF'], $currentEncoded)).'">'.\Image::getHtml('new.svg', $GLOBALS['TL_LANG'][$this->strTable]['move'][0]).'</a>';
 				}
+
+				if ($this->strPickerField)
```

Comment:
I checked this, too, but there are too many small differences to extract the code into a parent method.